src/main/java/com/example/demo/ConsoleReadWrite.java

```diff
@@ -0,0 +1,82 @@
+package com.example.demo;
+
+import com.example.demo.model.Match;
+import com.example.demo.model.Team;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Scanner;
+
+@Component
+@Getter
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class ConsoleReadWrite {
+
+    static final String YES = "y";
+    Tournament tournament;
+    Scanner scanner = new Scanner(System.in);
+
+    ConsoleReadWrite(Tournament tournament) {
+        this.tournament = tournament;
+    }
+
+    public void printMatches() {
+        System.out.println("\nRound, Team 1, Team 2, Score");
+        List<Match> matches = tournament.getMatchService().matchesToList();
```

Comment:
Spring beans should not be used by getter from another bean.
If you need that service, inject it here.